stone/backends/python_rsrc/stone_base.py

```diff
@@ -73,8 +73,8 @@ def __ne__(self, other):
     def __hash__(self):
         return hash((self._tag, self._value))
 
-    def _process_custom_annotations(self, annotation_type, processor):
-        # type: (typing.Type[T], typing.Callable[[T, U], U]) -> None
+    def _process_custom_annotations(self, annotation_type, context, processor):
```

Comment:
Since you are going to change context in `make_list_annotation_processor` and `make_map_value_annotation_processor`. I suggest we make it clear the the second parameter is the path.